pyannote/audio/utils/metric.py

```diff
@@ -177,13 +192,29 @@ def der_from_swf(
 
         # if (num_frames, num_speakers)-shaped, compute just one DER for the whole file
         if ndim == 2:
-            return self.compute_components_helper(hypothesis.data, reference.data)
+            if uem is not None:
```

Comment:
I usually prefer 

```
if something is None: 
   pass
else:
   pass
```

rather than 
```
if something is not None:
    pass
else:
    pass